templates/named.zone.erb

```diff
@@ -12,6 +12,11 @@ zone "<%= @zone %>" {
 <% if @zonetype == 'master' -%>
     update-policy {
             grant rndc-key zonesub ANY;
+            <%- unless @update_policy_rules.empty? -%>
```

Comment:
I think this line is redundant since iterating over an empty hash will not produce any output either.